R/module_tabs_with_filters.R

```diff
@@ -111,9 +111,10 @@ ui_tabs_with_filters <- function(id, modules, datasets) {
   filter_panel_btn <- tags$li(
     style = "flex-grow : 1;",
     tags$a(
+      id = "filter_hamburger",
       href = "javascript:void(0)",
       class = "menubtn",
-      onclick = "toggle_sidebar();",
+      onclick = "handleHamburgerClick();",
```

Comment:
What if we were more specific? I know this hamburger can be universal, but it will be used only for filter-panel and we would like to be more clear here

```suggestion
      onclick = "toggleFilterPanel();",
```

Reply:
I updated the name  in the js and here at the same time in the last commit